frontend/src/sketch.js

```diff
@@ -63,6 +63,15 @@ let radioBtn;
 // updates counter
 let updatesCounterElement;
 let updatesCounter = 0;
+//hotspots inputs
+let hotspotsCheckbox;
+let hotspotDestUpdateNumber;
+let hotspotLocUpdateNumber;
+let hotspotPassNumber;
+let positionRadBtn;
+let hotspotPositionX = 0;
+let hotspotPositionY = 0;
+
```

Comment:
Hotspots can't be at (0,0) by default because there might be a building there if we modify the map